galata/test/jupyterlab/collapsible-headings.test.ts

```diff
@@ -111,3 +111,188 @@ test.describe('Collapsible Headings; no_showHCB', () => {
     ).toMatchSnapshot('no_showHCB_expand_heading_via_collapser.png');
   });
 });
+
+async function populateNotebook2(page: IJupyterLabPageFixture) {
+  await page.notebook.setCell(0, 'markdown', '# Heading 1');
+  await page.notebook.addCell('code', '1+1');
+  await page.notebook.addCell('markdown', '## Heading 1.1');
+  await page.notebook.addCell('code', '2+2');
+  await page.notebook.addCell('markdown', '# Heading 2');
+  await page.notebook.addCell('code', '3+3');
+  await page.notebook.addCell('code', '4+4');
+}
+
+test.describe('Collapsible Headings; keyboard navigation', () => {
+  test.use({
+    mockSettings: {
+      '@jupyterlab/apputils-extension:themes': {
+        overrides: {
+          'content-font-family': 'Arial',
+          'content-font-size1': '12pt'
+        }
+      }
+    }
+  });
+
+  // create an empty notebook for each test
+  test.beforeEach(async ({ page }) => {
+    await page.notebook.createNew(fileName);
+  });
```

Comment:
It seems you run the those two actions for each test. So it could be moved here:

```suggestion
    await populateNotebook2(page);
    await page.notebook.run();
  });
```

Reply:
Thanks for the hint. Did all of that and similar streamlining in the first two tests I did previously.